ear.go

```diff
@@ -200,10 +426,144 @@ func (o *AttestationResult) Verify(data []byte, alg jwa.KeyAlgorithm, key interf
 // compatible with the requested signing algorithm.  On success, the complete
 // JWT token is returned.
 func (o AttestationResult) Sign(alg jwa.KeyAlgorithm, key interface{}) ([]byte, error) {
-	payload, err := o.ToJSON()
+	payload, err := o.MarshalJSON()
 	if err != nil {
 		return nil, err
 	}
 
 	return jws.Sign(payload, jws.WithKey(alg, key))
```

Comment:
Now that we have the claims-set as a map, we might want to call `jwt.Sign` instead of `jws.Sign` using `jwt.Token`'s `PrivateClaims`